client/App.js

```diff
@@ -1,14 +1,25 @@
 import React, { Component } from 'react';
-import { Route, Switch } from 'react-router-dom';
-import { Challenges, IndividualChallenge, Nav, Footer } from './components';
+import { HashRouter as Router, Route, Switch } from 'react-router-dom';
```

Comment:
Do we need to import HashRouter here? It should already be wrapped around the <App /> component in the index file.